appmanifest/META.yml

```diff
@@ -1,7 +1,6 @@
 spec: https://w3c.github.io/manifest/
 suggested_reviewers:
-  - anssiko
   - kenchris
   - marcoscaceres
   - mgiuca
-  - mounirlamouri
```

Comment:
This is outrageous! 😄

Reply:
heh, you really want to review tests?

Reply:
Sorry, I thought the 😄 would be a clue that it was a tongue in cheek reply 🙈

Reply:
just checking :) will spare you from this.